changes/fedekunze_2331-percent-votes

```diff
@@ -0,0 +1,2 @@
+[Changed] [#2287](https://github.com/cosmos/lunie/pull/2287) Use tags instead of commit hash @sabau
```

Comment:
ah here it is XD will the new changelog tool work with two lines @faboweb ?

Reply:
yes :)